src/test/attrs.test.js

```diff
@@ -105,17 +105,17 @@ describe('attrs', () => {
   })
 
   it('pass attrs to style block', () => {
-    /* Would be a React Router Link in IRL */
```

Comment:
hmm was this meant to mean "in real life"?

Reply:
Yeah I think so